app/gui.py

```diff
@@ -8,6 +8,8 @@
 
 from model import Eeg, Raw, bands
 
+raw_i=0
```

Comment:
issue: avoid global variables since this will be become rather difficult to maintain rather quickly. Instead move this to be a member variable of the RawPlotWindow class